### PR TITLE
Feature/cc 9 bugfix] price schema

### PR DESCRIPTION
**Pull request by Beryl-Zhou**
AC1: add cement floor price in the price schema

### CODE REVIEW

File: src/price/dto/price.dto.ts [hunk 1]
```diff
@@ -1,16 +1,19 @@
...
-  @IsString()
-  readonly tile_id: string;
-
-  @IsNumber()
-  @IsPositive()
-  readonly deliveryPrice: number;
+import { Type } from "class-transformer";
+import { IsArray, IsNumber, IsPositive } from "class-validator";
 
+export class TilesPrice {
```
Comment (chui-mel) [inline src/price/dto/price.dto.ts:4]: `TilePrice`?

File: src/price/dto/price.dto.ts [hunk 2]
```diff
@@ -1,16 +1,19 @@
...
+  tile_id: string;
+  tileName: string;
+  deliveryPrice: number;
+  tilePrice: number;
```
Comment (chui-mel) [inline src/price/dto/price.dto.ts:8]: `price`?

File: src/price/dto/price.dto.ts [hunk 3]
```diff
@@ -1,16 +1,19 @@
...
   @IsNumber()
   @IsPositive()
-  readonly tilePrice: number;
+  readonly cementFloorPrice: number;
 
-  readonly createdAt: Date;
-  readonly updatedAt: Date;
+  @IsArray()
+  @Type(() => TilesPrice)
+  readonly tilesPrice: TilesPrice[];
```
Comment (chui-mel) [inline src/price/dto/price.dto.ts:18]: `readonly tilePrices: TilePrice[]`

File: src/price/dto/price.dto.ts (hunk 1 above)
Comment (Beryl-Zhou) [reply src/price/dto/price.dto.ts]: updated~~

File: src/price/dto/price.dto.ts (hunk 2 above)
Comment (Beryl-Zhou) [reply src/price/dto/price.dto.ts]: updated~~

File: src/price/dto/price.dto.ts (hunk 3 above)
Comment (Beryl-Zhou) [reply src/price/dto/price.dto.ts]: updated~~

**Review by jennawan — APPROVED** [review]

**Review by tatoMa — APPROVED** [review]

**Pull request merged**